conda-recipe/meta.yaml

```diff
@@ -1,20 +1,23 @@
-{% set data = load_setup_py_data() %}
+{% set package_name = "pcdsdevices" %}
+{% set import_name = "pcdsdevices" %}
+{% set version = load_file_regex(load_file=os.path.join(import_name, "_version.py"), regex_pattern=".*version = '(\S+)'").group(1) %}
 
```

Comment:
Another small weird thing that isn't causing an issue in the gha conda build:

```
ClobberWarning: Conda was asked to clobber an existing path.
  source path: /home/runner/conda_pkgs_dir/libjpeg-turbo-2.1.4-h166bdaf_0/share/man/man1/rdjpgcom.1
  target path: /home/runner/micromamba/conda-bld/pcdsdevices_1673570096661/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/man/man1/rdjpgcom.1
```
https://github.com/pcdshub/pcdsdevices/actions/runs/3907251443/jobs/6676280720#step:11:1421